### DISCUSSION TITLE
Stupid method parameter popups in VSCode

### DISCUSSION THREAD

**Discussion by alyashraf97** [redhat-developer/vscode-java#3712] [Q&A] [answered]
![Screenshot from 2024-07-15 22-03-51 (copy)](https://github.com/user-attachments/assets/22dfe7c8-8042-4884-b79a-1df0259e62ff)

I don't know who's idea of a joke this was, but how do I disable this absolutely stupid and annoying popup?

**Comment by alyashraf97** [comment]
For information, I did try to google it and found no solution online for how to hide this stupid thing

settings.json:
`    "editor.parameterHints": false,
    "editor.parameterHints.enabled": false,
    "editor.suggest.snippetsPreventQuickSuggestions": false,
    "html.suggest.html5": false,
    "editor.snippetSuggestions": "none",
    "editor.quickSuggestions": {
        "other": false,
        "comments": false,
        "strings": false
    },
    "editor.hover.enabled": false,
    "editor.hover.above": false,
    "editor.hover.sticky": false,
    "editor.hover.hidingDelay": 1,
    "editor.hover.delay": 10000,
    "editor.parameterHints.cycle": false,
    "editor.suggest.showTypeParameters": false,
    "breadcrumbs.showTypeParameters": false,
    "outline.showTypeParameters": false,
    "editor.acceptSuggestionOnCommitCharacter": false,
    "java.inlayHints.parameterNames.enabled": "none"
`

**Comment by fbricon** [comment, marked as answer]
It's called Signature Help. You can toggle it was `java.signatureHelp.enabled`

**Reply by alyashraf97** [reply]
Thanks..
I must apologize for my antics, I spent over an hour trying to disable this feature instead of spending it doing anything useful.. There ought to be easier ways to reach an answer for these things instead of opening an issue on github.
